agent/command/git.go

```diff
@@ -787,7 +787,7 @@ func (c *gitFetchProject) fetchModuleSource(ctx context.Context,
 			if opts.token != "" {
 				errOutput = strings.Replace(errOutput, opts.token, "[redacted oauth token]", -1)
 			}
-			logger.Execution().Info(errOutput)
+			logger.Execution().Error(errOutput)
```

Comment:
I like your idea in the ticket of adding the module name. Should we maybe wrap the error and include module.Name?

Reply:
I'm maybe confused about how this prevents the output from being jumbled

Reply:
It doesn't make it not jumbled, but it made it easier to see with a red line in parsley...

I changed it now to collect in a buffer and display it one after another with every new line having the module name, and the error having the module name fast.